src/ReportGenerator.Core/Parser/Analysis/CodeFile.cs

```diff
@@ -136,6 +136,11 @@ internal CodeFile(string path, int[] lineCoverage, LineVisitStatus[] lineVisitSt
         /// <value>The number of total lines.</value>
         public int? TotalLines { get; private set; }
 
+        /// <summary>
+        /// Gets line coverage information for this file.
+        /// </summary>
+        public int[] LineCoverage => this.lineCoverage;
```

Comment:
@danielpalme I didnt see a way to expose Partially covered lines. Let me know how we can get partially covered lines and i can expose it in my next PR.

Reply:
The coverage status is tracked here:
https://github.com/danielpalme/ReportGenerator/blob/cc0c9df10b3db5e49248a545f07bf494ca9c3e29/src/ReportGenerator.Core/Parser/Analysis/CodeFile.cs#L46

I already exposed that field here:
https://github.com/danielpalme/ReportGenerator/blob/cc0c9df10b3db5e49248a545f07bf494ca9c3e29/src/ReportGenerator.Core/Parser/Analysis/CodeFile.cs#L148

The enum [LineVisitStatus](https://github.com/danielpalme/ReportGenerator/blob/master/src/ReportGenerator.Core/Parser/Analysis/LineVisitStatus.cs) contains the possible values: 

- NotCoverable
- NotCovered
- PartiallyCovered
-  Covered